src/world/zone_settings.h

```diff
@@ -0,0 +1,13 @@
+#ifndef SERVER_ZONE_SETTINGS_H
+#define SERVER_ZONE_SETTINGS_H
+
+#include "../common/cbasetypes.h"
```

Comment:
General note: (aside from one folder in the map/ai/ subfolder) we don't need to use relative paths anymore, `common/cbasetypes.h` will work where it didn't all the way back in DSP

Reply:
Good to know. Will make sure I don't add them in future locations. Removed this file anyway.